azure-pipelines.yml

```diff
@@ -296,4 +297,35 @@ jobs:
       - task: CmdLine@2
         displayName: 'Build Community Toolkit Sample'
         inputs:
-          script: 'dotnet build $(PathToCommunityToolkitSampleCsproj) -c Release'
\ No newline at end of file
+          script: 'dotnet build $(PathToCommunityToolkitSampleCsproj) -c Release'
+
+  - job: verify_dotnet_format
+    displayName: Verify Code Formatting
+    pool:
+      vmImage: macos-12
+    steps:
+      - task: CmdLine@2
+        displayName: 'Set Xcode Version'
+        inputs:
+          script: echo '##vso[task.setvariable variable=MD_APPLE_SDK_ROOT;]'/Applications/Xcode_$(XcodeVersion).app;sudo xcode-select --switch /Applications/Xcode_$(XcodeVersion).app/Contents/Developer
+      
+      - task: UseDotNet@2
+        displayName: Install Latest .NET SDK
+        inputs:
+          packageType: 'sdk'
+          version: '$(LATEST_NET6_VERSION)'
+
+      - task: CmdLine@2
+        displayName: 'Install Latest .NET MAUI Workload'
+        inputs:
+          script: dotnet workload install maui
+
+      - task: CmdLine@2
+        displayName: 'Verify Formatting'
+        inputs:
+          script: |
+            dotnet format $(PathToSamplesSolution) --verify-no-changes
+            status=$?
+            [ $status -eq 0 ] && echo "No errors found"
+            [ $status -ne 0 ] && echo "Formatting errors found. In the command line, please run \`dotnet format CommunityToolkit.Maui.Sample.sln\`, then commit + push the newly formatted code"
```

Comment:
Let's use variable instead of path to solution

Reply:
Done